rewrite-weblogic/docs/recipes/migrate-to-weblogic-141100.md

```diff
@@ -77,6 +80,9 @@ dependencies {
 ```
 2. Run `gradle rewriteRun` to run the recipe.
 
+> [!TIP]  
+> If you just want to dry run the recipe without changing the code, use `gradle rewrite:DryRun`.

```

Comment:
Same: The correct command is `gradle rewriteDryRun`. Reference: https://docs.openrewrite.org/reference/gradle-plugin-configuration#the-dryrun-task